packages/@okta/vuepress-site/docs/concepts/manage-user-creds/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Manage user credentials for your apps
+meta:
+  - name: description
+    content: Learn about user credentials and securely managing them
+---
+
+# Manage user credentials for your apps
+
+Securely managing your tokens is essential to mitigate risks that directly threaten the protection of user data, such as token theft and account takeover.
+
+## Terms
+
+* [Authorization server](/docs/concepts/auth-servers/): A server that mints OpenID Connect (OIDC) or OAuth 2.0 tokens. It applies access policies that determine who can be issued a token and what scopes that token can have. Conversely, the resource server applies access policies to decide which tokens and scopes it accepts in exchange for access to specific resources.
+
+  Okta supports two types of authorization servers:
+
+  * [Org authorization server](/docs/concepts/auth-servers/#org-authorization-server): A built-in server that supports simple SSO for OIDC apps or for obtaining access tokens for Okta APIs. It doesn't support custom scopes, claims, or policies.
+  * [Custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server): A server that allows you to create and apply authorization policies to secure your own APIs. You can also define custom scopes and claims, and integrate with API gateways. It's recommended to assign one authorization server per API product to maintain separate policies and token expiration times, and to prevent scope name collisions.
```

Comment:
```suggestion
  * [Custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server): A server that allows you to create and apply authorization policies to secure your own APIs. You can also define custom scopes and claims, and integrate with API gateways. Okta recommends assigning one authorization server per API product to maintain separate policies and token expiration times, and to prevent scope name collisions.
```